Source/Core/DolphinWX/AboutDolphin.cpp

```diff
@@ -78,8 +78,7 @@ AboutDolphin::AboutDolphin(wxWindow *parent, wxWindowID id,
 	wxTextCtrl* const Revision = new wxTextCtrl(this, wxID_ANY, RevisionText, wxDefaultPosition, wxDefaultSize, wxNO_BORDER | wxTE_READONLY | wxTE_NO_VSCROLL);
```

Comment:
This can likely be made into a static text as well. Then the BanishBackground function can be fully removed.